### PR TITLE
Update the README.md

### PR DESCRIPTION
**Pull request by pasindujw**
Update the README.md content to show proper installing steps.

---
This change is [Reviewable](https://reviewable.io/reviews/jenkinsci/github-plugin/201)

### CODE REVIEW

**Comment by pasindujw** [comment]
Hi,
Any feedback on this PR?

Thanks.

**Comment by KostyaSha** [comment]
The proper installing step is via plugin manager, this instructions needed only for dev builds, but for development there are generics docs, so this file looks redundant at all.

**Comment by KostyaSha** [comment]
@daniel-beck is there any general page that plugins could reference to instead of bundling into every plugin readme?

**Comment by pasindujw** [comment]
@daniel-beck kind reminder :)

**Comment by daniel-beck** [comment]
@KostyaSha @pasindujw Not really. There's user documentation on it, which (unsurprisingly) emphasizes the update center based install, but has a section on uploads: https://jenkins.io/doc/book/managing/plugins/#advanced-installation

I would recommend to add a redirect to jenkins.io if you're going to reference this a lot, that way links don't break when we reorg the site.

I doubt up to date developer docs have anything related, as between the above and `hpi:run` they're mostly covered.

File: README.md [hunk 1]
```diff
@@ -30,19 +30,13 @@ Jenkins Plugin Maven goals
...
-to create the plugin .hpi file.
-
-
-To install:
-
-1. copy the resulting ./target/rdoc.hpi file to the $JENKINS_HOME/plugins directory. Don't forget to restart Jenkins afterwards.
+2. copy the resulting ./target/rdoc.hpi file to the $JENKINS_HOME/plugins directory. Don't forget to restart Jenkins afterwards.
 	
-2. or use the plugin management console (http://example.com:8080/pluginManager/advanced) to upload the hpi file. You have to restart Jenkins in order to find the plugin in the installed plugins list.
+3. or use the plugin management console (http://example.com:8080/pluginManager/advanced) to upload the hpi file. You have to restart Jenkins in order to find the plugin in the installed plugins list.
```
Comment (daniel-beck) [inline README.md]: > You have to restart Jenkins in order to find the plugin in the installed plugins list.

This hasn't been true for a long time. _Updating_ requires restarts, _installing_ does not.

**Pull request closed without merging**